src/components/organisms/ExplorerPane/FilePane/FileSystemTree/FileSystemTree.tsx

```diff
@@ -27,6 +27,7 @@ const FileSystemTree: React.FC = () => {
   const [firstHighlightedFile, firstHighlightedFileRef] = useSelectorWithRef(state =>
     state.main.highlights.find(isFileSelection)
   );
+  const [fileMap, fileMapRef] = useSelectorWithRef(state => state.main.fileMap);
```

Comment:
`fileMap` is not used here so we can directly go for `const fileMapRef` = useRefSelector(...)